.travis.yml

```diff
@@ -31,31 +31,50 @@ jobs:
       - $SH "bundle exec rspec spec/tests/model_articulation_test/facility_spec.rb"
       - $SH "bundle exec rspec spec/tests/model_articulation_test/hvac_system_spec.rb"
       - $SH "bundle exec rspec spec/tests/model_articulation_test/loads_system_spec.rb"
+      - $SH "bundle exec rspec spec/tests/model_articulation_test/lighting_system_type_spec.rb"
       - $SH "bundle exec rspec spec/tests/model_articulation_test/service_hot_water_system_spec.rb"
       - $SH "bundle exec rspec spec/tests/model_articulation_test/site_spec.rb"
       # - $SH "bundle exec rspec spec/tests/model_articulation_test/weather_file_download_spec.rb"  // fails sometimes due to connection issues, so we exclude it here
       name: "Model Articulation Tests"
+    - script:
+        - travis_wait 40 $SH "bundle exec rspec spec/tests/model_articulation_test/hospital_occupancy_type_spec.rb"
```

Comment:
We are going to run out of travis credits pretty soon. Leave this as-is, but we will need to move over to github actions.

Reply:
we are already out...

Reply:
travis definitely fits in 2020